language/move-lang/tests/move_check/typing/module_call_visibility_script.move

```diff
@@ -0,0 +1,18 @@
+address 0x2 {
+
+module X {
+    fun f_private() {}
+    public fun f_public() {}
+    public(script) fun f_script() {}
+}
+
+module M {
+    use 0x2::X;
+    fun f_private_call_script() { X::f_script() }  // should fail
+    public fun f_public_call_script() { X::f_script() }  // should fail
+    public(script) fun f_script_call_script() { X::f_script() }  // should pass
+    public(script) fun f_script_call_public() { X::f_public() }  // should pass
```

Comment:
Move the passing ones into a separate test. I like to split all of the failing cases from all of the passing cases. Just makes it extra clear if anything changes